src/main/java/seedu/address/logic/commands/MassOpCommand.java

```diff
@@ -16,24 +18,28 @@
  */
 public class MassOpCommand extends Command {
 
-    public static final List<String> COMMAND_WORDS = List.of(new String[]{"mass", "Mass"});
+    //CHECKSTYLE.OFF: VisibilityModifier
+    public static List<String> commandWords = new ArrayList<String>(Arrays.asList("mass", "m", "Mass"));
+    //CHECKSTYLE.ON: VisibilityModifier
 
-    public static final String MESSAGE_USAGE = COMMAND_WORDS
+    public static final String MESSAGE_USAGE = commandWords
             + ":Tag or delete Tags people who are filtered by the filter in the person list.\n"
             + "Parameters:  \n"
-            + "Example: " + COMMAND_WORDS + "tag/delete <TagName> <filter command>";
+            + "Example: " + commandWords + "tag/delete <TagName> <filter command>";
```

Comment:
this should be "tag/delete_tag" instead since the right syntax for delete a tag is delete_tag